pkg/helpers/config/userdata.yaml

```diff
@@ -1,13 +1,13 @@
 #cloud-config
-repo_update: true
-package_update: true
-packages:
-  - docker 
 runcmd:
-  - sudo service docker start
+  # taking care manually of the packages to be able to make it non-blocking
+  - sudo yum -y update || echo "didn't update package"
+  - sudo yum -y install docker || echo "docker not installed by yum"
+  # Redirecting the output because of failed start with Golden AMI (should be improved in OSD-12345)
+  - sudo service docker start 2>1 > /dev/null || echo "docker not started by systemctl"
```

Comment:
we should embed this to AMI. Also we can add `sudo docker pull ${VALIDATOR_IMAGE}` part to AMI to get a faster validation check